pkg/cloudclient/aws/private.go

```diff
@@ -47,7 +47,7 @@ var (
 		"me-south-1":     "",
 	}
 	// TODO find a location for future docker images
-	networkValidatorImage string = "quay.io/bngoy/osd-network-verifier:1"
+	networkValidatorImage string = "quay.io/twilliams725/osd-network-verifier:2"
```

Comment:
Pushed a new image based on these changes, this updates the project to use the new image until we have better automation around image CD in place.